rules/DB_processing/format_n_train_classifiers.rules

```diff
@@ -1,37 +1,35 @@
-## Rules to format the taxonomy into a format for DADA2 implementation of RDP.
```

Comment:
and you're un-commenting here what you commented it out in the previous commit. That makes reviewing the commits painful...
There is no reason to keep this as a separate commit, why not squash the commits together?